.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@wildnihal Make sure this file is included in `.gitignore` so that this doesn't get commited to this repo.